server/settings/prod.py

```diff
@@ -38,7 +38,7 @@
 WTF_CSRF_CHECK_DEFAULT = True
 WTF_CSRF_ENABLED = True
 
-MAX_CONTENT_LENGTH = 15 * 1024 * 1024  # Max Upload Size is 10MB
+MAX_CONTENT_LENGTH = 20 * 1024 * 1024  # Max Upload Size is 20
```

Comment:
Why is this 20 but the upload is 15MB? 

15mb per file - 20mb total.  I'd support more but I'm not sure if we would timeout first.